task/rpm-ostree/0.1/rpm-ostree.yaml

```diff
@@ -173,6 +174,7 @@ spec:
           -e COMMIT_SHA="$COMMIT_SHA" \
           --rm \
           -v "$BUILD_DIR/workspaces/source:$(workspaces.source.path):Z" \
+          -v "$BUILD_DIR/.docker/:/root/.docker:Z" \
```

Comment:
I'm a bit confused by this one, is it really necessary?  It seems to expose the pull secret to the container itself, but I don't think we need that here FWIW.

Reply:
Oh good catch, this is a different use case than the buildah-remote task. This one wants to use the credentials to pull the BUILDER_IMAGE itself, and there's no need to pass the credentials to that image again

I think we need to rsync the docker credentials to the $HOME of the VM user

Reply:
You need to either sync the credentials to the $HOME/.docker dir, or explicitly tell podman which credentials file to use.